data/mods/Animatronics/animatronics.json

```diff
@@ -1,552 +1,878 @@
 [
     {
-        "type" : "MONSTER",
-        "id" : "mon_animatronic_fox",
-        "name": "animatronic fox",
-        "name_plural": "animatronic foxes",
-        "species":"ROBOT",
-        "symbol":"F",
-        "color":"red",
-        "size":"LARGE",
-        "material":"steel",
-        "diff":12,
-        "aggression":100,
-        "morale":100,
-        "speed":120,
-        "melee_skill":6,
-        "melee_dice":4,
-        "melee_dice_sides":6,
-        "melee_cut":6,
-        "dodge":8,
-        "armor_bash":12,
-        "armor_cut":8,
-        "item_chance":80,
-    "death_drops": {
-        "groups": [
-            [ "robots", 60 ],
-             [ "default_zombie_clothes", 40 ],
-            [ "child_items", 75 ]        ]
+	"type" : "MONSTER",
+	"id" : "mon_animatronic_fox_old",
+	"name": "old animatronic fox",
+	"species":"ROBOT",
+	"symbol":"F",
+	"color":"red",
+	"size":"LARGE",
+	"material":"steel",
+	"diff":12,
+	"aggression":100,
+	"morale":100,
+	"speed":140,
+	"melee_skill":6,
+	"melee_dice":4,
+	"melee_dice_sides":6,
+	"melee_cut":10,
+	"dodge":6,
+	"armor_bash":6,
+	"armor_cut":4,
+	"vision_day":30,
+	"vision_night":5,
+	"item_chance":80,
+	"death_drops": {
+	"groups": [
+		[ "robots", 60 ],
+			[ "default_zombie_clothes", 40 ],
+			[ "child_items", 75 ]
+		]
     },
-    "luminance":0,
-        "hp":180,
-        "special_freq":[0],
-        "death_function":"BROKEN",
-        "special_attack":["NONE"],
-        "description":"A bipedal animatronic fox, nearly twice your size.  Its teeth and claws look quite sharp.  Though it is a robot, it stinks like death.",
-        "flags":["SEES", "GRABS", "HEARS", "BASHES", "ATTACKMON", "ELECTRONIC", "NO_BREATHE", "VIS50"]
+	"luminance":0,
+	"hp":100,
+	"special_freq":[0],
+	"death_function":"BROKEN",
+	"special_attack":[["PARROT", 0]],
+	"description":"A bipedal animatronic fox, nearly twice your size.  Its teeth and claws look quite sharp.  Its right had is replaced with a hook.  It looks heavily worn and degraded.",
```

Comment:
hand

Might want to condense that?  "[...]size.  Its right paw has been replaced by a hook, but said hook, the claws on the left paw, and teeth all look quite sharp, despite its overall worn-down appearance."